testsuite/src/it/java/org/graalvm/tests/integration/DebugSymbolsTest.java

```diff
@@ -161,12 +167,14 @@ public void debugSymbolsSmokeGDB(TestInfo testInfo) throws IOException, Interrup
             try (BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(process.getOutputStream()))) {
                 Logs.appendlnSection(report, appDir.getAbsolutePath());
                 Logs.appendln(report, String.join(" ", processBuilder.command()));
+                final long increasedTimeoutMs = (UsedVersion.getVersion(false)
+                        .compareTo(Version.create(23, 0, 0)) >= 0) ? CMD_LONG_TIMEOUT_MS : CMD_DEFAULT_TIMEOUT_MS;
                 boolean result = waitForBufferToMatch(report, stringBuffer,
                         Pattern.compile(".*Reading symbols from.*", Pattern.DOTALL),
-                        3000, 500, TimeUnit.MILLISECONDS);
+                        increasedTimeoutMs, 500, TimeUnit.MILLISECONDS); // Time unit s the same for timeout and sleep.
```

Comment:
Once concern I have with having a global timeout is that we might miss slow downs that affect different tests. E.g., at the moment the only affected test (that requires much longer time to start) is the quarkus-microprofile, by increasing the timeout for all tests we might miss the opportunity to notice a slow down in other tests.

Reply:
That is false, it's all Quarkus tests, i.e. both Microprofile and Vert.x one.
+1 for making it more fine grained when we get it working at least with what we have.